types.cc

```diff
@@ -3057,6 +3057,17 @@ bytes serialize_for_cql(const abstract_type& type, collection_mutation_view v, c
     });
 }
 
+bytes serialize_field_index(uint16_t idx) {
+    bytes b(bytes::initialized_later(), sizeof(uint16_t));
```

Comment:
Let's add protection against that user that creates a UDT with 65536 fields and then complains that the wrong fields gets updated.

Reply:
Done, but I should add a test for that.

Reply:
And done